app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -28,10 +28,10 @@
     private final int heightPixels;
     private final int widthPixels;
     private final boolean candyWordFirst;
-    private final String nidoFemale;
-    private final String nidoMale;
+    private final String nidoFemale = "Nidoran♀";
+    private final String nidoMale = "Nidoran♂";
```

Comment:
No, these names can be translated. The plan was to fetch them from the pokemon array rather than duplicating it.